api/src/org/apache/cloudstack/api/command/user/snapshot/CreateSnapshotCmd.java

```diff
@@ -224,7 +224,7 @@ public void execute() {
 
         Snapshot.LocationType locationTypeToReturn = Snapshot.LocationType.values()[0];
 
-        if (locationType != null && locationType >= 0 && locationType <= Snapshot.LocationType.values().length) {
+        if (locationType != null && locationType >= 1 && locationType <= Snapshot.LocationType.values().length) {
```

Comment:
Looks good.

Maybe you can update the description of locationType to change the word 'primary' to 'standard', too. Thanks